packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -103,6 +105,7 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 				getMiniflareOptions(normalizedPluginConfig, viteConfig, viteDevServer),
 			);
 
+			let routerWorker = await getRouterWorker(miniflare);
```

Comment:
Is caching this object important?
Could we just call `getRouterWorker()` on each request inside the middleware?

Reply:
I'm not sure how much of an advantage there is over doing this on every request. This would also mean calling `createMiddleware` on each request, which seems a bit wasteful. We can do that if you prefer though.

Reply:
Good point. That was my mistake. I'll change to go with this approach then (I presume it's very fast for Miniflare to get the Worker stub).

Reply:
f2b1fd9